operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -199,6 +257,21 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
                 LOG.finest("The request specified a '" + START_QUERY_PARAMETER + "' and/or '" + END_QUERY_PARAMETER + "' query parameter. They are not valid for resource type '" + compartmentType + "', so will be ignored.");
                 searchParameters = queryParametersWithoutDates;
             }
+            
+            // Need to get the value of the _type parameter out, if it exists
```

Comment:
can we do this just once (before the loop that starts on 254) instead of on each one?

Reply:
No longer needed after our conversation.  Removed the code starting on line 261.